src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -60,12 +61,30 @@
           :error-handler (flash-message/default-error-handler :top description)})
    {}))
 
-(defn- to-create-catalogue-item []
+(rf/reg-event-db
+ ::set-selected-items!
+ (fn [db [_ items]]
+   (assoc db ::selected-items items)))
+
+(rf/reg-sub
+ ::selected-items
+ (fn [db _]
+   (::selected-items db)))
+
+(defn- create-catalogue-item-button []
   [atoms/link {:class "btn btn-primary"}
    "/administration/catalogue-items/create"
    (text :t.administration/create-catalogue-item)])
 
-(defn- to-catalogue-item [catalogue-item-id]
+(defn- change-form-button [items]
+  [:button.btn.btn-primary
+   {:disabled (when (empty? items) :disabled)
+    :on-click (fn []
+                (rf/dispatch [:rems.administration.change-catalogue-item-form/enter-page items])
+                (navigate! "/administration/catalogue-items/change-form") )}
```

Comment:
Why is enter-page dispatched here in the link instead of the router, like all other pages?

Reply:
If it's because of the items parameter, that could also be transmitted as query parameters in the URL.

Reply:
We **don't** want to reset everything on page enter i.e. when you change URL. That would disallow view of single item and back while having something selected.

Reply:
Query parameters would introduce a lot of duplication or coupling between the table where everything is selected and change form page.